cirq-google/cirq_google/engine/engine_client.py

```diff
@@ -170,7 +171,9 @@ async def _run_retry_async(self, func: Callable[[_M], Awaitable[_R]], request: _
                 message = err.message
                 # Raise RuntimeError for exceptions that are not retryable.
                 # Otherwise, pass through to retry.
-                if err.code.value not in RETRYABLE_ERROR_CODES:
```

Comment:
The http.HTTPStatus is IntEnum so it works in the `x in [int1, int2,...]` expression just like plain integer.
Please replace with
```python
if err.code not in RETRYABLE_ERROR_CODES:
```